RELEASECHECKLIST.md

```diff
@@ -3,16 +3,24 @@
 ### Pre-Build
 * Update AssemblyInfo for kOS project
 * Update AssemblyInfo for kOS.Safe project
-* Update Resources\GameData\kOS\kOS.version
-* Update doc\source\conf.py
-* Update CHANGELOG.MD
+* Update AssemblyInfo for kOS.Safe.Test project
+* Update `Resources\GameData\kOS\kOS.version`
+* Update `doc\source\conf.py`
+* Update `CHANGELOG.MD`
+* Update `doc\source\changes.rst`
```

Comment:
I just did a test and github DOES allow single-backtick to work in its MD files, but NOT in its comments (like this one) even though it claims both are using its Markdown.  I have no idea why it's inconsistent about this, but the point is, I don't have to change the single backticks after all.